back/models/services.js

```diff
@@ -12,4 +12,14 @@ Service.getAllServices = (callback) => {
 	});
 };
 
+// add new service
+Service.addNewServiceClient = (serviceInfo, callback) => {
+	connection.query('INSERT INTO client_service SET date=?, client_id=?, service_id=?',
```

Comment:
A data (se é data de registo) deve ser adicionada automaticamente pela BD.
Exemplo:
`created datetime NOT NULL DEFAULT CURRENT_TIMESTAMP`